tests/test_cli.py

```
@@ -13,4 +13,6 @@ def test_cli(self):
             "init_reaction",
             "autotest",
         ):
-            sp.check_output(["dpgen", subcommand, "-h"])
+            output = sp.check_output(["dpgen", subcommand, "-h"])
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)